### PR TITLE
Fix/autest conditions

### PR DESCRIPTION
**Pull request by dragon512**
add HasATSFeature API
add HasCurlFeature API
add version check for which version of Autest is used
some fixes for the via test
add feature test for ats
Update documentation

### CODE REVIEW

**Comment by zwoop** [comment]
[approve ci autest]

**Comment by dragon512** [comment]
looks like autest was installed on python 2.7

I would 
pip uninstall autest
pip3 install autest --upgrade

**Comment by zwoop** [comment]
[approve ci autest]

File: tests/getting_started.md [hunk 1]
```diff
@@ -193,3 +193,59 @@ ts.Disk.remap_config.AddLine(
...
+ * TS_USE_CERT_CB
+ * TS_USE_SET_RBIO
+ * TS_USE_TLS_ECKEY
+ * TS_USE_LINUX_NATIVE_AIO
+ * TS_HAS_SO_PEERCRED
+ * TS_USE_REMOTE_UNWINDING
+ * TS_HAS_128BIT_CAS
+ * TS_HAS_TESTS
+ * TS_HAS_WCCP
+ * SPLIT_DNS
```
Comment (zwoop) [inline tests/getting_started.md]: What an unfortunate naming :). Outside of the scope for this PR, but might be worth fixing this in a separate PR.

**Comment by zwoop** [comment]
Remember to set the Milestone too :)

**Pull request merged**